packages/strapi-plugin-graphql/services/GraphQL.js

```diff
@@ -560,7 +561,7 @@ module.exports = {
 
               switch (association.nature) {
                 case 'manyToMany': {
-                  const arrayOfIds = obj[association.alias].map(related => {
+                  const arrayOfIds = (obj[association.alias] || []).map(related => {
```

Comment:
use _.map instead

Reply:
Thank you for highlighting this!

I'm avoiding to use Lodash methods as much as possible. ECMAScript is going better and better, I'm pretty sure we could remove this dependency in the future. Moreover, the performances are similar on a small array like this.